t/test-rest-api.php

```diff
@@ -319,6 +319,41 @@ function test_endpoint_get_entries() {
 
 	/**
 	 * Integration test
+	 * Test accessing the crud endpoint with an insert action
+	 */
+	function test_endpoint_crud_action() {
+
+		// Create an author and set as the current user
+		$author_id = $this->factory->user->create( array(
+			'role' => 'author',
+		) );
+		wp_set_current_user( $author_id );
+
+		// Create a post
+		$this->factory->post->create();
+
+		// The POST data to insert
+		$post_vars  = $this->build_entry_args( array(
+			'crud_action' => 'insert',
+		));
+
+		// Try to access the endpoint and insert an entry
+		$request  = new WP_REST_Request( 'POST', self::ENDPOINT_BASE . '/1/crud/' );
```

Comment:
I'm making an assumption here that after the table is truncated and 1 record is inserted, that the ID of that record will be 1. But, that could be a dangerous assumption.
I have that ID of 1 hard coded in a few spots in the tests and that should be changed to grab the actual ID that was just inserted.